stylesheets/components/cards.css

```diff
@@ -0,0 +1,151 @@
+.card {
+  border: 1.35px solid var(--dark-pastel-gray);
+  border-radius: 8px;
+  background-color: var(--white-color);
+  gap: 1rem;
+  max-width: 18.5rem;
+  position: relative;
+}
+.card:hover {
+  box-shadow: 4px 4px 15px var(--charcoal-grey);
+  transition: 0.4s ease-in;
+}
+.card-header {
+  max-height: 17rem;
+}
+.card-img {
+  border-top-left-radius: 8px;
+  border-top-right-radius: 8px;
+}
+.icon-top-right {
+  color: var(--dark-pastel-gray);
+  position: absolute;
+  top: 10px;
+  right: var(--spacing-xs);
+}
+.icon-dismiss {
+  color: var(--charcoal-grey);
+}
+.card-body {
+  margin: 0 1rem 1rem 1rem;
+  gap: 0.25rem;
+}
+.card-title {
+  font-size: 1.4rem;
+}
+.card-text {
+  color: var(--text-color);
+  font-size: 0.9rem;
+  font-weight: 300;
+}
+.card-section {
+  justify-content: space-between;
+  flex-wrap: wrap;
+}
+.card-price {
+  gap: var(--spacing-xs);
+}
+.card-current-price {
+  font-size: 1.2rem;
+  font-weight: 600;
+}
+.card-price-striked {
+  text-decoration: line-through var(--gravel-color);
+}
+.card-rating {
+  gap: 0.3rem;
+}
+.rating-icon-wrapper {
+  gap: 0.1rem;
+}
+.rating-icon {
+  color: var(--yellow-dark);
+  font-size: 0.7rem;
+}
+.rating-count {
+  font-size: 0.7rem;
+}
+.card-actions {
+  margin-top: 1.5rem;
+  gap: 1rem;
+}
+.quantity {
+  border: 1px solid var(--dark-pastel-gray);
+  border-radius: 3px;
+}
+.quantity p,
+.quantity button {
+  padding: 0.3rem 0.6rem;
+}
+.quantity button:hover {
+  background-color: var(--dark-pastel-gray);
+}
+.add-to-cart {
+  background-color: var(--avocado-green);
+  color: var(--white-color);
+  border-radius: 3px;
+  gap: 0.3rem;
+  padding: 0 0.5rem;
+  font-size: 0.9rem;
+}
+.add-to-cart:hover {
+  background-color: var(--spruce-color);
+}
+.btn-add-to-cart {
+  font-weight: 600;
+}
+.badge {
```

Comment:
You can use the badge component here.

Reply:
Didn't use them because the positioning and stylings were a bit different here